app/models/techie.rb

```diff
@@ -3,6 +3,12 @@
 class Techie < ApplicationRecord
   belongs_to :user
   has_one :person, through: :user
+  has_many :requests, foreign_key: :point_person_id
 
   delegate :name, to: :person
+  delegate :email, to: :person
```

Comment:
This can be combined into the above:

```suggestion
  delegate :name, :email, to: :person
```